Jasmine/Common/Helpers/Random.swift

```diff
@@ -14,6 +14,15 @@ enum Random {
         return from + Int(arc4random_uniform(UInt32(toInclusive - from + 1)))
     }
 
+    /// Generates a random integer
+    ///
+    /// - Parameters:
+    ///   - toInclusive: end of range, inclusive
+    /// - Returns: a random number from 0 to toInclusive
+    static func integer(toInclusive: UInt) -> Int {
```

Comment:
@riwu If you wanted this behavior, just add a default `0` to existing implementation. No need to use UInt.
```
static func integer(from: Int = 0, toInclusive: Int) -> Int {
   assert(toInclusive > from, "`toInclusive` must be larger than `from` number")
   return from + Int(arc4random_uniform(UInt32(toInclusive - from + 1)))
}
```